### PR TITLE
Defines gql addParkingPlace mutation

### PR DESCRIPTION
**Pull request by yulduzetta**
Defines gql `addParkingPlace` mutation
 // TODO for the _**next**_ PR: move price prop from ParkingPlace model to Inventory

### CODE REVIEW

**Comment by yulduzetta** [comment]
Clean state:
image

**Comment by yulduzetta** [comment]
**2. Add parking:**
image

**Comment by yulduzetta** [comment]
**3. User with parking places:**
image

File: server/schemas/resolvers.js [hunk 1]
```diff
@@ -38,12 +36,17 @@ const resolvers = {
...
-        //   return await Inventory.create(parkingLoation._id,args.startDate ,args.endDate );
-        // }
+        const parkingLot = await ParkingPlace.create({
+          ...args,
+          provider: context.user._id,
+        });
+        await User.findByIdAndUpdate(
+          { _id: context.user._id },
+          { $push: { parkingPlace: parkingLot._id } }
+        );
```
Comment (bhumisha) [inline server/schemas/resolvers.js:47]: Here ParkingPlace  is getting used to User table than we don't need providerID in ParkingPlace table. We might need to look model again.

**Review by bhumisha — APPROVED** [review]
lgtm

**Pull request merged**